netci.groovy

```diff
@@ -276,6 +276,44 @@ def buildArchConfiguration = ['Debug': 'x86',
 }
 
 // **************************
+// Define outerloop testing for netfx that can build and run.  Run locally on each machine.
+// **************************
+[true, false].each { isPR ->
```

Comment:
Why does this need its own group as opposed to be included with the other outerloop runs?